src/SriServiceProvider.php

```diff
@@ -27,12 +29,9 @@ public function boot()
             __DIR__.'/../config/subresource-integrity.php' => config_path('subresource-integrity.php'),
         ]);
 
-        Blade::directive('mixSri', function ($arguments) {
-            return "<?php echo app('".Sri::class."')->mix({$arguments}) ?>";
-        });
-
-        Blade::directive('assetSri', function ($arguments) {
-            return "<?php echo app('".Sri::class."')->asset({$arguments}) ?>";
-        });
```

Comment:
Do we no longer want to support Laravel 6.x (which is still in LTS)? The view class components only came in v7.x.

Reply:
v3 will drop the support for older Laravel version.

Since Laravel 8 was just released, I'm still undecided to drop Laravel 7 as well 🤔

Reply:
Hmmm... I see. All good then.